client/src/components/Form.js

```diff
@@ -0,0 +1,123 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import isValidBrofile from '../lib/validatFunction';
+/*import { useLocalStorage } from '../hooks/useLocalStorage';*/
+
+export default function Form({ submitFunction, open }) {
+
+    const userBrofile = {
+        bro_username: '',
+        bro_name: '',
+        bro_location: '',
+        image: ''
+    }
+    const [newBrofile, setNewBrofile] = useState(userBrofile);
+    const [valid, setValid] = useState(false)
+
+    const handleChange = event => {
+        const field = event.target;
+        const value = field.value;
+        setNewBrofile({
+            ...newBrofile,
+            [field.name]: value
+        })
+
+        function submitForm(event) {
+            event.preventDefault();
+            if (isValidBrofile(newBrofile)) {
+                setValid(true);
+                submitFunction(newBrofile);
+                setNewBrofile(userBrofile);
+            }
+        }
+        return (
+            <>
```

Comment:
No need for the additional React Fragment as your <FormWrapper> wraps the whole content